packages/dashmate/docs/installation.md

```diff
@@ -0,0 +1,42 @@
+# Installation
+
+This guide provides instructions for installing Dashmate, a distribution package for Dash node installation.
+
+## Dependencies
+
+Before installing Dashmate, ensure you have the following dependencies installed:
+
+* [Docker](https://docs.docker.com/engine/installation/) (v20.10+)
+* [Node.js](https://nodejs.org/en/download/) (v20, NPM v8.0+)
+
+For Linux installations, you may optionally wish to follow the Docker [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to manage Docker as a non-root user. Otherwise, you will have to run CLI and Docker commands with `sudo`.
+
+## Installing Dashmate
+
+Use NPM to install dashmate globally in your system:
+
+```bash
+$ npm install -g dashmate
+```
```

Comment:
I thought the recommendation was to not install via npm anymore. Has that changed?

Reply:
Good point! Updated